hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/ha/SCMHAInvocationHandler.java

```diff
@@ -88,7 +88,9 @@ private Object invokeLocal(Method method, Object[] args)
    */
   private Object invokeRatis(Method method, Object[] args)
       throws Exception {
-    long startTime = Time.monotonicNowNanos();
+    LOG.trace("Invoking method {} on target {}", method, ratisHandler);
+    // If we do want to track time taken by ratis,
+    // it should be done via a metric and not via Info logging.
```

Comment:
Thanks @sky76093016.
In the comment, can you add a TODO so that it can be tracked.
Something like this:

> // TODO: Add metric here to track time taken by Ratis

Reply:
Of course.